### DISCUSSION TITLE
It does not work that terminating group task

### DISCUSSION THREAD

**Discussion by youguanxinqing** [celery/celery#7019] [Q&A]
Hi! I want to terminate a **group** task when a subtask failed. My code is as follow：
```python
class ASummaryTask(Task, abc.ABC):
    def on_failure(self, exc, task_id, args, kwargs, einfo):
        celery_app.control.revoke(self.request.group, terminate=True, wait=True)
        print("失败")

    def on_success(self, retval, task_id, args, kwargs):
        print("成功")


@celery_app.task(bind=True)
def group_task(self):
    promise = group([sub_task.s(i=i) for i in range(20)], propagate=True)
    promise.apply_async()
    return "ok"


@celery_app.task(bind=True, base=ASummaryTask, propagate=True)
def sub_task(self, i):
    time.sleep(5)
    if i == 3:
        raise ValueError("3 error")

    return f"sub task ok {i}"
```

But `celery_app.control.revoke(self.request.group, terminate=True, wait=True)`  dosen't work. Or group task isn't able to be terminated ?

What could I do for this?  Hope gotten some helps and thanks in advance!

**Comment by beni1028** [comment]
Were you able to find a solution to this problem?
I am having the same issue as well

**Comment by zicokuo** [comment]
me too,
ex. SCTID20230215163654907190757994708�(GroupsID)
it not work with
celery.current_app.control.revoke(gid)
or
celery.current_app.backend.delete_group(gid)
